oat-sa/tao-core
# [FEATURE][SOLAR-59] Portal authoring access

Ticket: https://oat-sa.atlassian.net/browse/SOLAR-59

## What's Changed

- Env Var `REDIRECT_AFTER_LOGOUT_URL` that configures the logout URL after redirect

- Introduced new Theme for the portal flow
- new feature flag `FEATURE_FLAG_TAO_AS_A_TOOL` which allows to enable of new theme.
- Env Var `PORTAL_URL`  that configures the main URL for the portal

## TODO 

- [x] Update package.json and package-lock.json after release of https://github.com/oat-sa/tao-core-ui-fe/pull/563


## Dependencies PRs

- [[TAO Portal](- [tao-portal](https://github.com/oat-sa/tao-portal/pull/221)
- https://github.com/oat-sa/tao-core-ui-fe/pull/563

## How to test

1. login via portal with an user which has content developer access
2. click on the content back tile
3. you should see the new theme.
